views/index.pug

```diff
@@ -1,22 +1,27 @@
 doctype html
 head
-	title CrewLink Server
+	title #{name}
 	link(rel="icon", href="https://github.com/ottomated/CrewLink/raw/master/logo.png", type="image/x-icon")
-	meta(property='og:title' content='CrewLink Server')
+	meta(property='og:title' content='RMDash.fr CrewLink Server')
```

Comment:
Please do not change the project's branding.

```suggestion
	meta(property='og:title' content='CrewLink Server')
```

Reply:
Sorry, I thought I had reverted my specific site's changes when I moved it to the new branch. Must have missed this one. It has been corrected via 8b29d79